lib/cisco_node_utils/cmd_ref/interface_channel_group.yaml

```diff
@@ -17,17 +17,33 @@ channel_group:
   default_value: false
   nexus:
     get_value: '/^channel-group (\d+)$/'
-    set_value: '<state> channel-group <group> <force>'
   ios_xr:
     get_value: '/^bundle id (\d+).*$/'
     set_value: '<state> bundle id <group>'
 
+channel_group_set:
+  set_value: '<state> channel-group <group> <force> mode <mode>'
+
 description:
   kind: string
   get_value: '/^description (.*)/'
   set_value: '<state> description <desc>'
   default_value: ''
 
+feature_lacp:
```

Comment:
Better to put `feature_lacp:` into the feature specific yaml/api file
* https://github.com/tomcooperca/cisco-network-node-utils/blob/c4609185d3a5d9afc4a7eec20ef8e00d6e91f7bc/lib/cisco_node_utils/cmd_ref/feature.yaml
* https://github.com/tomcooperca/cisco-network-node-utils/blob/c4609185d3a5d9afc4a7eec20ef8e00d6e91f7bc/lib/cisco_node_utils/feature.rb